libvast/src/system/node.cpp

```diff
@@ -331,7 +325,10 @@ caf::message dump_command(const invocation& inv, caf::actor_system&) {
         }
       },
       [=](caf::error& err) mutable {
-        request_error = std::move(err);
+        request_error = caf::make_error(ec::unspecified,
+                                        "'dump' failed to get taxonomies from "
+                                        "type-registry: {}",
+                                        std::move(err));
```

Comment:
This seems to be missing a call to `fmt::format()`

Also, not related to this PR, but rust's anyhow has this cool concept of adding additional context to an error. Since we use this pattern a lot I wonder if we could implement a helper that works in spririt like this:

```
  request_error = err.context("while trying to get taxonomies from type-registry in 'dump'");
```

Reply:
We actually have an epic internally for error contexts and rendering errors more nicely, but we've just never gotten around to doing it.

The call to format is definitely missing, nice catch.

I was thinking about poisoning `caf::make_error` and introducing our own `vast::make_error` that always goes through {fmt}. That would avoid this rather common error.